src/Console/Command/TweeterCommand.php

```diff
@@ -177,15 +186,19 @@ protected function generateTweet($event)
     protected function shortenUrl($url)
     {
         $google_key = getenv('GOOGLE_SHORTEN_KEY');
+
         $link = new Link;
         $link->setLongUrl('http://mremi/url-shortener');
-        $googleProvider = new GoogleProvider(
-            $google_key,
-            array('connect_timeout' => 1, 'timeout' => 1)
-        );
 
+        $options =  [
+            'connect_timeout' => 1,
+            'timeout' => 1,
+        ];
+
+        $googleProvider = new GoogleProvider($google_key, $options);
```

Comment:
Should this be changed to 

`$googleProvider = new GoogleProvider(getenv('GOOGLE_SHORTEN_KEY'), $options);`